src/java_10thEdition/AskMe.java

```diff
@@ -0,0 +1,57 @@
+package java_10thEdition;
+
+import java.sql.SQLOutput;
+import java.util.Random;
+
+interface SharedConstants{
+    int NO = 0;
+    int YES = 1;
+    int MAYBE = 2;
+    int LATER = 3;
+    int SOON = 4;
+    int NEWER = 5;
+}
+class Question implements SharedConstants{
+    Random rand = new Random();
+    int ask(){
+        int prod = (int)(100* rand.nextDouble());
+        if(prod<30)
+            return NO;
+        else if (prod<60)
+            return YES;
+        else if (prod<75)
+            return LATER;
+        else if (prod<98)
+            return SOON;
+        else return NEWER;
+    }
+}
+ class AskMe implements SharedConstants{
+    static void ansver(int result){
+        switch (result) {
```

Comment:
What happens if the result value doesn't have any of provided cases? For example result = 100000
Your method should be versatile

Reply:
if I understood this program correctly, (I did not understand the Ansver method works at all) the next line
--int prod = (int)(rand.nextInt(100));--
in class Question transmits numbers from 0 to 100 maximum, 100000 cannot be here.
but it is not exactly

Reply:
Do you see my comment?

Reply:
Yes, you are right.
But let's imagine that someone else will pass such a value to the method.
You just need to add default case